docs/tools/vdb_table/data/couchbase.json

```diff
@@ -0,0 +1,181 @@
+  },
+  "full_text": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/c-sdk/current/concept-docs/full-text-search-overview.html",
+    "comment": ""
+  },
+  "embeddings_text": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external embedding models"
+  },
+  "embeddings_image": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "embeddings_structured": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "via external embedding models"
+  },
+  "rag": {
+    "support": "partial",
+    "source_url": "https://docs.couchbase.com/cloud/vector-search/run-vector-search-ui.html#large",
+    "comment": "via external LLMs"
+  },
+  "recsys": {
+    "support": "",
+    "source_url": "",
+    "comment": ""
+  },
+  "langchain": {
+    "support": "full",
+    "source_url": "https://python.langchain.com/v0.1/docs/integrations/vectorstores/couchbase/",
+    "comment": ""
+  },
+  "llamaindex": {
+    "support": "full",
+    "source_url": "https://docs.llamaindex.ai/en/stable/api_reference/readers/couchbase/",
+    "comment": ""
+  },
+  "managed_cloud": {
+    "support": "full",
+    "source_url": "https://www.couchbase.com/products/capella/",
+    "comment": ""
+  },
+  "pricing": {
+    "value": "Couchbase Pricing",
+    "source_url": "https://www.couchbase.com/pricing",
+    "comment": ""
+  },
+  "in_process": {
+    "support": "none",
+    "source_url": "",
+    "comment": ""
+  },
+  "multi_tenancy": {
+    "support": "full",
+    "source_url": "",
+    "comment": "via collections and RBAC"
+  },
+  "disk_index": {
+    "support": "full",
+    "source_url": "",
+    "comment": ""
+  },
+  "ephemeral": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/buckets.html#replication-dcp-and-xdcr",
+    "comment": ""
+  },
+  "sharding": {
+    "support": "full",
+    "source_url": "https://docs.couchbase.com/server/current/learn/buckets-memory-and-storage/vbuckets.html",
+    "comment": "Both internode and intranode"
+  },
+  "doc_size": {
+    "bytes": 20000000,
+    "unlimited": false,
+    "source_url": "https://docs.couchbase.com/server/current/learn/views/views-writing-views.html#limits-on-document-sizes-for-indexing",
+    "comment": ""
+  },
+  "vector_dims": {
+    "value": 2048,
+    "unlimited": false,
+    "source_url": "https://docs.couchbase.com/server/current/vector-search/create-vector-search-index-ui.html#procedure",
+    "comment": "Increasing to 4096 in pending 7.6.2 release"
+  },
+  "int8_quantization": {
+    "support": "partial",
+    "source_url": "",
+    "comment": "Automatic algorithm selection between: IdMap2,Flat, IVF,Flat, IVF,SQ8"
+  },
+  "github_stars": {
+    "value": 6000,
+    "source_url": "https://github.com/couchbase",
+    "comment": "",
```

Comment:
this field is only for repositories which are open-source and contain the vector search functionality within them